src/jit/importer.cpp

```diff
@@ -3208,7 +3201,7 @@ GenTreePtr      Compiler::impIntrinsic(CORINFO_CLASS_HANDLE     clsHnd,
         {
             NO_WAY("JIT must expand the intrinsic!");
         }
-        else if (IsIntrinsicImplementedByUserCall(intrinsicID))
+        else if ((retNode->gtFlags & GTF_CALL) != 0)
```

Comment:
I am not sure that i understood why we we need the above change.  In fact GTF_CALL flag is set on retNode only is IsIntrinsicImplementedByUserCall(IntrinsicID) is true for RyuJIT.  Is CoreRT is using other than RyuJIt as backends right now?